README.md

```diff
@@ -1,53 +1,88 @@
-# Wiki
+# Publish to GitHub wiki action
 
-This action takes the contents of a provided directory and synchronizes it with your
-repository's wiki. This allows, for example, pull requests for the wiki.
+📖 Deploy docs from your source tree to the GitHub wiki
 
-The goal is to expose the wiki to PRs, so that potential contributors who aren't
-collaborators can contribute to the wiki and go through a review process. Alternatively,
-one can disabled the "Restrict editing to collaborators only" setting for their wiki,
-but this allows the community to make unreviewed, potentially unwanted, changes.
+## Features
 
-By default, it will use the `.github/wiki` directory.
+🌐 Works across repositories (with a [PAT]) \
+📚 Pretty interface for Markdown docs \
+⤴️ Lets you open PRs for wiki docs
 
-## Before using this action
+## Usage
+
+❗ Make sure you turn on the GitHub wiki feature in your repo's settings menu.
+You'll also need to _manually_ create a dummy page to initialize the git repo
+that powers the GitHub wiki. If you don't, when we push to `your-repo.wiki.git`,
```

Comment:
This is the first step, so it's logical that this text is here. But, in order of priority, I think it makes sense to have the YAML code block first. All users will find the code block useful, but only users who haven't initialized their wiki need to read this.

Reply:
Sounds good. 👍 I can also make the image smaller or do `<img align=right src=...>` to make it float right.

Like this:

---

<img align=right src="https://placekitten.com/200/200" />

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Sodales neque sodales ut etiam sit amet nisl. Sed euismod nisi porta lorem mollis. Nunc congue nisi vitae suscipit tellus. Lacus sed viverra tellus in hac habitasse. Semper auctor neque vitae tempus quam pellentesque nec. Elit scelerisque mauris pellentesque pulvinar. Blandit aliquam etiam erat velit scelerisque in. Mattis molestie a iaculis at erat. Et netus et malesuada fames ac turpis. Quam quisque id diam vel quam. Nisl tincidunt eget nullam non nisi est sit. Vulputate ut pharetra sit amet aliquam id diam. Sed viverra tellus in hac.

Reply:
Aligning right sounds good, should result in less vertical scrolling :+1: